music_assistant/providers/bbc_sounds/__init__.py

```diff
@@ -0,0 +1,405 @@
+
+                elif queue.state in [PlayerState.PAUSED, PlayerState.IDLE]:
+                    self.logger.debug("Queue is paused, reverting to station information")
+                    await self._display_station(queue.queue_id, station_id)
+
+    @property
+    def supported_features(self) -> set[ProviderFeature]:
+        """Return the features supported by this Provider."""
+        return SUPPORTED_FEATURES
+
+    @property
+    def is_streaming_provider(self) -> bool:
+        """
+        Return True if the provider is a streaming provider.
+
+        This literally means that the catalog is not the same as the library contents.
+        For local based providers (files, plex), the catalog is the same as the library content.
+        It also means that data is if this provider is NOT a streaming provider,
+        data cross instances is unique, the catalog and library differs per instance.
+
+        Setting this to True will only query one instance of the provider for search and lookups.
+        Setting this to False will query all instances of this provider for search and lookups.
+        """
+        return True
+
+    async def get_radio(self, prov_radio_id: str) -> Radio:
+        """Get full radio details by id."""
+        station = await self.client.stations.get_station(prov_radio_id, include_stream=True)
+        if not station or not station.stream:
+            raise MusicAssistantError("No valid radio stream found")
+        return Radio(
+            item_id=prov_radio_id,
+            provider=self.domain,
+            name=f"BBC {station.name}",  # main title on details page
+            uri=station.stream.uri,
+            metadata=MediaItemMetadata(
+                description="",  # subtitle on details page
+                label=station.stream.show_title,
+                images=UniqueList(
+                    [
+                        MediaItemImage(
+                            type=ImageType.THUMB,
+                            path=station.logo_url,
+                            provider=self.lookup_key,
+                            remotely_accessible=True,
+                        )
+                    ]
+                ),
+            ),
+            provider_mappings={
+                ProviderMapping(
+                    item_id=prov_radio_id,
+                    provider_domain=self.domain,
+                    provider_instance=self.instance_id,
+                )
+            },
+        )
+
+    async def get_stream_details(self, item_id: str, media_type: MediaType) -> StreamDetails:
+        """Get streamdetails for a track/radio."""
+        station = await self.client.stations.get_station(item_id, include_stream=True)
+        if not station or not station.stream:
+            raise MusicAssistantError(f"No stream found for {item_id}")
+
+        return StreamDetails(
+            stream_title=station.description,
```

Comment:
My guess is that this is the reason the initial play displays 'bbc_station' on the initial start. I think you want to use the logic from `_display_station` here to properly calculate the stream title on the first playback.